### ISSUE TITLE
Provider doesn't handle state of appcatalyst-daemon 

### ISSUE THREAD

**Issue by mootpt** [vmware/vagrant-vmware-appcatalyst#11]
Appcatalyst provider does not work out of the box and requires the appcatalyst-daemon to be started before vagrant will function properly. This behavior is unexpected for most, as the existing VMware providers don't require a separate daemon to be running for `vagrant up` to function as expected:

```
vagrant up --provider=vmware_appcatalyst        
Can't connect to AppCatalyst, please verify connectivity.
```

Ideally, the daemon should be handled by the provider and/or this requirement should be documented.

**Comment by frapposelli** [comment]
Thanks for reporting this @mootpt

It's something that we're considering adding, both from the AppCatalyst standpoint (starting up the daemon on boot, having a menubar helper to start/stop the daemon) and from the Vagrant provider (adding a `do you want to start the daemon? [y/n]` prompt).

**Comment by mootpt** [comment]
Thanks @frapposelli.

**Comment by Xe** [comment]
It is apparently [unvagranty](https://github.com/mitchellh/vagrant/issues/2662) to prompt the user for any input on the provisioning of a machine. If the user is wanting to use the appcatalyst stack it makes sense to do one of two things:
1. Fix it for the user and tell them that it was fixed.
2. Explode with a [better error message](https://gist.github.com/Xe/e9db837644aa4cbd3ccb) to tell the user how to fix it.

**Comment by Xe** [comment]
This issue should have been closed by #20, oops.
